src/js/quote/quote.js

```diff
@@ -0,0 +1,57 @@
+// import api function
+
+function getQuoteExpireTime() {
+  let tomorrow = new Date();
+  tomorrow.setDate(tomorrow.getDate() + 1);
```

Comment:
Can be simplified to 
`const tomorrow = new Date(new Date().getTime() + 24 * 60 * 60 * 1000)`